.env

```diff
@@ -10,3 +10,7 @@ HOST=127.0.0.1
 # CONFIGURABLE OPTIONS:
 PORT=3333
 AWS_PROFILE_NAME=default
+AWS_S3_BUCKET_NAME="fayez-private-bucket"
+AWS_CLOUDFRONT_ACCESS_KEY_ID="K2AMMADKCTQS8B"
+AWS_CLOUDFRONT_PRIVATE_KEY="-----BEGIN RSA PRIVATE KEY-----\nMIIEowIBAAKCAQEAxuREKOvCv6u9L8NqId782Zaz1EBQy6xlhtx/gOUki0bpyVp8\nl99DASeX+jkCvCHKBw2DJIHBp0mxjWNEUsp9OX03depRypkXwbmEjt1MBrxRlgB0\nNIu0A3jdEjtFMjdxxYggmSZiJNXo7ifC+jevG5yvHzOudnOVbrVyXKGmV5sBewGw\nEvr3EzL/90a78KK9AAG4kvg7xDv/dmJkORTFBIa8odVirXvwdIxhyRQEej+FT2NV\n1VMrx26b4iaZE6wQVmfemISX4n+x5gS0az9BM4ZXdSUqQNpHq9adeSJjd7BO6DsC\n2O8ei7obcu/qPnnrzGxKb75ClPa2vnhsQGqsXQIDAQABAoIBAQCSuTSbJ4mM0ooH\nYVa7VAAeGLHjA+tWxYBEZAffs6so4BB5zToa48ZPZMEWA9bDqP2cBFjZpquQ2KmW\nD0QI+34z2e7/GvRpzFSdQlnK9vAuC+D4Eq+nT+Rlp1UXLQhYClTxDwJVWN6QJAnp\nHwYUht20NftuvjaXBZ+Yg1yxf0LyXodTmELpkheQEes1PrnpxugRKpnVRpcCUvow\nONmYn5IcqnBas0UIevMsLZf7539bFHt35cILeO9Xgp2ufJwEg3H28zLTqYb8v3Y1\npC3xKityun/rXstFZB1iTPPria3yayuxV/9wXGYHi/SNhFZO8gI5dNYPdvPbUhOk\nG17PZSNVAoGBAOZAMe5Qz/gDGqL8ZP6R15njXfiId7tHGW4sF1bPVSlRxKnNuuz1\niesXh0DpH6tqanNv7/s+RGSUahzKULgq1CGad1yCPqx5RV1g47C1ioYeOpKMGlov\nZFS7b6RfTLdDdU1zK/da3dWm970JwgkphNDBdX5oEDN+NzPkT1Aqu/4fAoGBAN0i\nTDF5c3FMKj49yQAZ5+KEYdJJ1tNfbK9cGak9HZYS3ESgtigzRG4CQ5fjnbmHhcdd\nN7CC/q/7wNiIBS6Q8BYILUPJP9e7sM6mYT2LvJ3/fGVBf2VYbErAA6WyJ4FS+8mW\nH2sZGi8EVss/ihwCT9c+E68faYtmP5aPNYLbjw4DAoGAHknBh7TvtpNJ9Li43Ff5\nAZBMMz5YabVPnhOEunGppTbRZPjgJv2ht+AvdZV1CgqEmEzYR7eILabsUn82oRNZ\nNlZoby3e6rdFLaRqLzuiTkQ5tj9h1A056H+Djx7OK6lfC+IjnXVBizJckmxvLt9c\nTk20k/1CbFOlFVlqKYSFmEcCgYAcCvhsocFBEe6dfApcZ+yfLjiViWWj4osO5fgf\nHS/+Ux/52AOhr3azGLhnr6bV21jcwnaIfOQ9fwiirQUzNZAsyxaqbQAapKO0YScd\nvfXKd5bX5Gl6ea8p/5W20pPt848YrmAHneRTKBK1rbYe/TAQ1L98StiApBPWTQz1\nLlDC7wKBgH0fWZ9rFOvQN3Wu+TUmJ/Jx4WZKN/u8d8bB5iLlP29gEfTmWJW8yYKC\nTOwyUgyfYnXqXgKBvGERzNWdGtTgXCu7T4eMr7Alu4wzFqd/Lkw/leBYaTlJtiaZ\n1e7IttCvD4P033mdN/XszCaU8w87MgUJH8tZC1bEoO0WFAvwQm6q\n-----END RSA PRIVATE KEY-----"
```

Comment:
This is restricted to back-end code only, wont be shown in the front-end, I pushed this here so it can be tested fast. Also, I can make use of the Prefix URL only instead of using public/private keys

Reply:
Even for back-end code, we cannot embed private keys into code, especially that this repo is public. Secrets are typically handled differently in backend. We don't want encryption, so let's just use the requested URL prefix, which is also more flexible in not relying on CloudFront as a CDN.